docs/tutorials/adding_source.md

```diff
@@ -1,11 +1,8 @@
 # Adding a source
 
-## 1. Add a source
-To add a source, create a new file in `viadot/sources`. The source must inherit from the `Source` base class and accept a `credentials` parameter.
+## Add a source
+To add a source, create a new file in `viadot/sources`. The source must inherit from the `Source` or `SQL` class and have credentials implemented using Pydantic.
```

Comment:
Can you add more details and an example?

Reply:
[📝 Added examples in adding_source.md](https://github.com/dyvenia/viadot/pull/657/commits/1e5e5968f7f16ea9ff3897c6e4bcc81d5fe8a2de) 

I added a link to the article and existing sources. I think that in this article it is explained in a fairly accessible way, so I think it will be enough. Can it be like that ?

Reply:
[📝 Improved link in adding_source.md](https://github.com/dyvenia/viadot/pull/657/commits/249bbb5d58de8ac89fe1cc582c4b126abf01125d)

Reply:
No, I think this is the tutorial for creating sources, so this is the place this should be explained in. The pydantic docs are too generic and examples are only part of the explanation -- it's good to include them, but not as a link but as a code snipped within the explanation.

Reply:
[📝 Improved add a source tutorial](https://github.com/dyvenia/viadot/pull/657/commits/f94e819d1ee5e0110708250472975e5f905b57ca)

Added simple tutorial how to create source and imporved description abot it how to add tests
[📝 Improved add unit test description](https://github.com/dyvenia/viadot/pull/657/commits/904f7341080099b2c234f06292c2e6a7fdb64c34)